.nvmrc

```diff
@@ -0,0 +1 @@
+v14.16.0
```

Comment:
Putting on my 'community volunteer leader' hat: Should we add nvm / properly setting node version stuff to the readme? I think most people are using docker these days but some vols do still do the individual steps, either as a learning exercise or because of the intensity of docker.

I'll also node that we have `.ruby-version` in gitignore, so we'll probably want to follow a similar pattern there?

Reply:
Hello itme one of (the only?) not-docker-runner 😄  

I definitely am onboard with both committing our `.nvmrc` this and getting this into the setup. If anything I'd be a fan of removing `.ruby-version` from the ignore file so we can play nicer with rbenv for the bare metal setup.

Reply:
Please feel free to push to this branch. I don't have strong opinions about this, and also don't know what would be best for the community in this case.

Reply:
I'll kick up a PR for this in a sec.

Reply:
https://github.com/DCAFEngineering/dcaf_case_management/pull/2309 is my attempt at setup instructions for nvm / keeping our bare metal instructions up to date. if that looks chill to you both I think we're good.